tests/test_stories.py

```diff
@@ -9,27 +9,24 @@
 import json
 
 poi1 = {
-    '_id' = 1
     'name': 'Himalayan Chimney',
-    'date': 'Date(2018, 2, 27)',
+    'date': '2018-02-02',
     'description': 'Yum',
     'map_year': 2018,
     'x_coord': 12,
     'y_coord': 43
 }
 
 poi2 = {
-    '_id' = 2
     'name': 'Mount Everest',
-    'date': 'Date(2018, 5, 15)',
+    'date': '2018-05-15',
     'description': 'Snow',
     'map_year': 2018,
     'x_coord': 15,
     'y_coord': 30
 }
 
 poi3 = {
-    '_id' = 3
     'name': 'Tampa Bay',
     'date': 'Date(2018, 8, 3)',

```

Comment:
the date field is kinda weird with tests, but might as well make this consistent with the other pois